build.gradle

```diff
@@ -143,5 +140,5 @@ spotless {
 // generate ANTLR visitor and listener classes from grammar
 generateGrammarSource {
     maxHeapSize = "64m"
-    arguments += ["-visitor", "-listener" ]
+    arguments += ["-visitor", "-listener"]
```

Comment:
Hier war ich wohl zu extrem beim Überfliegen. Das hier bitte nicht

Reply:
Das ist der Auto-Formatierung von IJ (ctrl+alt+l) geschuldet, und war zuvor nicht formatiert

Reply:
Ja kann passieren nur IJ kann jeder konfigurieren wie er Lust und Laune hat. Wenn jetzt der nächste wieder die Datei neu formatiert, findet man früher oder später nicht mehr wer für diese Reihe zuständig war. Wenn du das ok hast das deine IJ Formattierung ist, die wir verwenden, dann kannst du hier einfach resolven.

Reply:
der punkt ist, dass man in diesem fall IJ nicht so konfigurieren kann, dass zeile 146 nicht unformatiert wäre. also ist so oder so unformatierte konfiguration eingecheckt worden, und der fehler lag nicht bei mir.